queries.sql

```diff
@@ -109,12 +109,12 @@ JOIN animals A ON A.id = Vi.animal_id
 WHERE V.name = 'Stephanie Mendez'
 GROUP BY A.name, V.name, Vi.date_of_visit;
 
-SELECT V.name AS vet, COUNT(A.name)
-FROM visits Vi
-JOIN vets V ON V.id = Vi.vet_id
-JOIN animals A ON A.id = Vi.animal_id
-WHERE V.name = 'Stephanie Mendez'
-GROUP BY V.name;
+-- SELECT V.name AS vet, COUNT(A.name)
+-- FROM visits Vi
+-- JOIN vets V ON V.id = Vi.vet_id
+-- JOIN animals A ON A.id = Vi.animal_id
+-- WHERE V.name = 'Stephanie Mendez'
+-- GROUP BY V.name;
```

Comment:
- [ ] Unless the commented code help to the reader we should remove those lines of code that we are not gonna use, remember YAGNI.

Reply:
done